crates/openapi/src/routes/customers.rs

```diff
@@ -0,0 +1,101 @@
+/// Customers - Create
+///
+/// Creates a customer object and stores the customer details to be reused for future payments. Incase the customer already exists in the system, this API will respond with the customer details.
```

Comment:
Split this into multiple lines. Ideally one line should not have more than 100 characters.